src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,27 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private ExecutorService executor = Executors.newFixedThreadPool(5);
+    private List<EventListener> listenersList = new CopyOnWriteArrayList<>();
 
+    public void registerListener(EventListener listener) {
+        listenersList.add(listener);
     }
 
     public void deregisterListener(EventListener listener) {
-
+        listenersList.remove(listener);
     }
 
     public void notifyEvent(Event event) {
```

Comment:
It's good practice to ensure that the event handling logic is exception-safe. You might want to wrap the `listener.onEvent(event)` call inside a try-catch block to prevent one listener's failure from affecting the others.